backend/BalanceBuddy/src/main/java/com/example/balancebuddy/config/JWTToUserConvertor.java

```diff
@@ -9,9 +9,14 @@
 import org.springframework.security.authentication.UsernamePasswordAuthenticationToken;
 import org.springframework.security.oauth2.jwt.Jwt;
 
+// Converts a JWT (JSON Web Token) into a UsernamePasswordAuthenticationToken which is designed for simple
+// presentation of a username and a password
 @Component
 public class JWTToUserConvertor implements Converter<Jwt, UsernamePasswordAuthenticationToken> {
 
+    // This method extracts the user ID from the JWT's subject field, creates a MyUser object and encapsulates
+    // it in a UsernamePasswordAuthenticationToken with no authorities. This token is then used by Spring Security
+    // to authenticate the user in the application

```

Comment:
Same for this comment. You can just write a short summary of what this methods does/returns based on some JWT token.